src/RestaurantButton.js

```diff
@@ -0,0 +1,11 @@
+import React, { useState } from "react";
+
+function  RestaurantButton (props) {
+  return (
+    <button onClick={props.orderOne} className="btn btn-primary">
```

Comment:
nice use of passing in an onClick handler to a component - you'll use this pattern a lot